code/modules/modular_computers/file_system/computer_file.dm

```diff
@@ -36,4 +37,16 @@ var/global/file_uid = 0
 	else
 		temp.filename = filename
 	temp.filetype = filetype
-	return temp
\ No newline at end of file
+	temp.password = password
+	return temp
+
+/datum/computer_file/proc/can_access_file(var/mob/user, input_password = "")
+	if(!password)
+		return TRUE
+	else
+		if (!input_password)
```

Comment:
We don't have spaces between the `if` keyword and the bracket for conditional statements in our style guide.